languages/csharp/exercises/concept/enums/.docs/instructions.md

```diff
@@ -1,27 +1,36 @@
+# Instructions
+
 In this exercise you'll be processing log-lines.
 
-Each log line is a string formatted as follows: `"[<LEVEL>]: <MESSAGE>"`.
+Each log line is a string formatted as follows: `"[<LVL>]: <MESSAGE>"`.
 
-There are three different log levels:
+These are the different log levels:
 
-- `INFO`
-- `WARNING`
-- `ERROR`
+- `Unknown`
+- `Trace`
+- `Debug`
+- `Info`
+- `Warning`
+- `Error`
+- `Fatal`
```

Comment:
These log levels directly refer to the log level that was introduced in the text above, and that refers to the log level in the file, _not_ the enum. Please change it to:

```suggestion
- `TRC` (trace)
- `DBG` (debug)
- `INF` (info)
- `WRN` (warning)
- `ERR` (error)
- `FTL` (fatal)
```